app/mikane/src/app/services/message/success/success-message.component.ts

```diff
@@ -2,6 +2,7 @@ import { Component, Inject } from '@angular/core';
 import { MAT_SNACK_BAR_DATA } from '@angular/material/snack-bar';
 
 @Component({
+	selector: 'success-message-component',
```

Comment:
Where is this used? 🤔

Reply:
Nowhere, just required for fixing this, apparently 🤔 
![Screenshot 2023-09-21 175858](https://github.com/pudding-tech/mikane/assets/5494439/e4a604c0-0717-42d8-832e-35fe6c34fb37)